backend/tracim_backend/lib/mail_notifier/notifier.py

```diff
@@ -129,6 +130,7 @@ def notify_content_update(self, content: Content):
         except Exception as e:
             # TODO - G.M - 2018-08-27 - Do Better catching for exception here
             logger.error(self, 'Exception catched during email notification: {}'.format(e.__str__()))
+            logger.error(self, traceback.format_exc())
```

Comment:
Please use https://docs.python.org/2/library/logging.html#logging.Logger.exception instead, it print traceback automatically.

Reply:
@buxx  I wanted to do so, but logger use here, is actually not a logging logger, but a specific tracim object from tracim v1 (/lib/utils/logger.py). need to check how to add logger.exception feature in it .

Reply:
@inkhey Okay. I think you can add it with:

``` diff
diff --git a/backend/tracim_backend/lib/utils/logger.py b/backend/tracim_backend/lib/utils/logger.py
index 0b8b4af..6044e01 100644
--- a/backend/tracim_backend/lib/utils/logger.py
+++ b/backend/tracim_backend/lib/utils/logger.py
@@ -48,5 +48,10 @@ class Logger(object):
             Logger.TPL.format(cls=self._txt(instance_or_class), msg=message)
         )
 
+    def exception(self, instance_or_class, message):
+        self._logger.exception(
+            Logger.TPL.format(cls=self._txt(instance_or_class), msg=message)
+        )
+
 
 logger = Logger('tracim')
```